src/components/ContextMenu/ContextMenu.styled.ts

```diff
@@ -32,3 +34,45 @@ export const StyledDot = styled.span`
   background-color: ${mainFontColor};
   border-radius: 50%;
 `;
+
+export const StyledDropDown = styled.div`
+  width: ${dropDownWidth};
```

Comment:
No action needed, just for further reading. There are various recommendation to organization of your HTML/CSS code, I personally follow these recommendations written by Bootstrap creator
https://codeguide.co/#css-syntax